open-sphere-base/analysis/src/main/java/io/opensphere/analysis/table/functions/ColumnFunctionFactory.java

```diff
@@ -0,0 +1,45 @@
+package io.opensphere.analysis.table.functions;
+
+import java.util.List;
+
+import io.opensphere.core.Toolbox;
+import io.opensphere.core.util.collections.New;
+
+/**
+ * A factory use to instantiate column functions.
+ */
+public class ColumnFunctionFactory
```

Comment:
I think this is a registry not a factory.

Reply:
pe·dan·tic
========
/pəˈdan(t)ik/
*adjective*
 : narrowly, stodgily, and often ostentatiously learned

*ex: a pedantic insistence that we follow the rules exactly*

Reply:
Also, done.

Reply:
Factory method pattern : Define an interface for creating an object, but let subclasses decide which class to instantiate. The Factory method lets a class defer instantiation it uses to subclasses.

Registry : A well-known object that other objects can use to find common objects and services.